docs/docs/using-onivim/command-line.md

```diff
@@ -9,3 +9,34 @@ sidebar_label: Command Line
 ### Getting help
 
 ### Managing extensions
+
+By default, user extension are loaded from the following paths:
+
+- Windows: `%LOCALAPPDATA%/Oni2/extensions` 
+- OSX & Linux: `~/.config/oni2/extensions`
+
+This can be overridden via the `--extensions-dir`, like:
+
+```
+oni2 --extensions-dir /some/path/with/extensions
+```
+
+### Logging
+
+The `-f` argument can be used to keep Onivim 2 attached to the terminal,
+and provide logging output.
```

Comment:
Would it be a good idea to mention `ONI2_DEBUG` and `ONI2_LOG_FILE` here as well?

Reply:
Would be a good place for it!